gradle/testing/randomization/policies/solr-tests.policy

```diff
@@ -100,6 +100,9 @@ grant {
   permission java.lang.RuntimePermission "loadLibrary.jaas";
   permission java.lang.RuntimePermission "loadLibrary.jaas_unix";
   permission java.lang.RuntimePermission "loadLibrary.jaas_nt";
+  // needed by ONNX integration (TODO: there is a cleaner way to handle this)
+  permission java.lang.RuntimePermission "loadLibrary.*"; // TODO: make more specific
```

Comment:
above there's `${common.dir}` and `${common-solr.dir}` usage and i imagine some property usage here could avoid the top-level wildcard use, but not yet looked into details; illustration:
```suggestion
  permission java.lang.RuntimePermission "loadLibrary.${someBuildDirectory}/*";
```